src/hooks/useCategorizeChannels.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+
+import { useChannelStore } from "@/store/useChannelStore";
+
+const useCategorizeChannels = () => {
+  const radioChannelList = useChannelStore((state) => state.radioChannelList);
+  const interestChannelIds = useChannelStore(
+    (state) => state.interestChannelIds
+  );
+  const [favoriteChannelList, setFavoriteChannelList] = useState([]);
+  const [otherChannelList, setOtherChannelList] = useState([]);
+
+  useEffect(() => {
+    if (radioChannelList.length === 0) {
```

Comment:
```suggestion
    if (!radioChannelList.length) {
```

0이 falsy한 값이라 이렇게 리팩토링 할 수 있을 것 같은데 두 분은 어떤 조건이 가독성 측면에서 보기 괜찮은가요? @faber-h @Eun0713

Reply:
저는 재윤님이 말씀해주신 `if (!radioChannelList.length)` 형태가 더 간결하고 읽기 편한 것 같아요 :) 좋은 방식인 것 같습니다!

Reply:
두 분 모두 좋은 의견 감사해요!

`radioChannelList.length === 0`가 혼동 없이 직관적인 코드라 명확하게 작성하고자 했는데
가독성 측면에서 두 분의 방식도 좋은 것 같아요!

반영했습니다~